MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -5697,6 +5696,22 @@ public string GetInputModifier(EncodingJobInfo state, EncodingOptions encodingOp
 
             inputModifier = inputModifier.Trim();
 
+            // Apply -probesize if configured
+            var probeSizeArgument = string.Empty;
+            var ffmpegProbeSize = _config.GetFFmpegProbeSize();
+
+            if (!string.IsNullOrEmpty(ffmpegProbeSize))
+            {
+                probeSizeArgument = $"-probesize {probeSizeArgument}";
+            }
+
+            if (!string.IsNullOrEmpty(probeSizeArgument))
+            {
+                inputModifier += $" {probeSizeArgument}";
+            }
+
+            inputModifier = inputModifier.Trim();
```

Comment:
This should be inside the if. There's no reason to trim if probeSizeArgument is null.